### PR TITLE
Judging

### PR DESCRIPTION
**Pull request by ofpau**
## What this PR does / Why we need it
Add Judging app for reviewing projects.

This is expected to provide the following features:

- [x] Import projects from devpost. 
- [x] Assign numbers (with order and location) to projects for presentations.
- [x] See status of presentations. 
- [x] Show presentation scores in admin view.  

## Some questions
- [x] I have read the contributing guidelines
- [x] I abide by this repository Code of Conduct
- [x] I understand that my PR won't be merged until Travis gives a "green light"

### CODE REVIEW

File: app/templates/base.html [hunk 1]
```diff
@@ -108,6 +108,7 @@
                                     <a href="{% url 'check_in_list' %}">Check-in</a></li>
                             {% endif %}
                         {% endif %}
+                        <li><a href="{% url 'import_projects' %}">Judging</a></li>
```
Comment (casassg) [inline app/templates/base.html]: Remember to check if it needs to show it as activate.

**Comment by casassg** [comment]
Try adding `__init__.py` in `judging/`

**Comment by ofpau** [comment]
I think this is ready to merge now. Please ~roast~ review :smile:

**Comment by oriolclosa** [comment]
Haven't seen anything yet, but amazing! XD

**Comment by ofpau** [comment]
Just fixed the bug that you found @oriolclosa.

There's probably a nicer way to have the right views shown to the right users, sorry but I can't think of anything better at this moment. We can take some time to rethink this part after the hackathon.

**Pull request merged**